### ISSUE TITLE
ACCESS-ESM1-5-tasmax.yaml fails with ssp585 cleaning validation from missing time steps

### ISSUE THREAD

**Issue by brews** [ClimateImpactLab/downscaleCMIP6#344]
ACCESS-ESM1-5-tasmax.yaml fails with ssp585 cleaning validation. Blocking progress on run #187 

This is from https://argo.cildc6.org/workflows/default/e2e-tasmax-jobs-46qf2.

Logs with the error show:

```
Validating gs://clean-b1dbca25/cmip6/ScenarioMIP/CSIRO/ACCESS-ESM1-5/ssp585/r1i1p1f1/day/tasmax/gn/v20210318.zarr
Traceback (most recent call last):
  File "/argo/staging/script", line 18, in <module>
    _test_timesteps(ds, data_type, time_period)
  File "/opt/dodola/dodola/core.py", line 645, in _test_timesteps
    assert (
AssertionError: projection cmip6 file is missing timesteps, only has 108405
```

**Comment by brews** [comment]
Paging Dr. @dgergel

**Comment by dgergel** [comment]
Eee I'm guessing it's missing timesteps - I'll check in a bit

**Comment by dgergel** [comment]
@brews interesting case - ssp585 for ACCESS-ESM1-5 actually has output through 2300. I haven't seen this before. 

Seems like we should update the slicing step if the ssp output extends beyond 12/31/2100 and trim it?

**Comment by brews** [comment]
Closed with PR #365
